tests/results/modal.html

```diff
@@ -0,0 +1,15 @@
+<div aria-labelledby="modal_title_id-label" class="fade modal modal-class-ex" id="modal-id-ex" role="dialog"
```

Comment:
`aria-labelledby="modal_title_id-label"`

I'm afraid this here is a accessibility failure. Wave is a little vague in the failure, axe gives a better explanation. 

> To solve this problem, you need to fix at least (1) of the following:
    aria-label attribute does not exist or is empty
    aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
    Element has no title attribute

It seems to me that this is at least failing on "references elements that do not exist", where the div is labelled by `modal_title_id-label` yet there is no element with that id.

Reply:
I will update the code to make this `aria-labelledby="modal_title_id"`. Since the div with `modal_title_id` exists and will satisfy the ARIA for this element.